### ISSUE TITLE
Addons list is not updated in README (9.0 and 10.0 branch)

### ISSUE THREAD

**Issue by elicoidal** [OCA/business-requirement#75]
Currently addons list is not updated in the README.md for branches v9.0 and v10.0.
It is updated properly in v8.0.

Is there a specific setup?

**Comment by sbidoul** [comment]
You need to add the placeholder in README.md. It will then be updated nightly.

```
[//]: # (addons)
this part will be replaced nightly
[//]: # (end addons)
```

**Comment by elicoidal** [comment]
fixed with #73 #74
